samples/constitutions/default/actions.js

```diff
@@ -144,7 +144,13 @@ function checkJwks(value, field) {
 }
 
 function checkX509CertBundle(value, field) {
-  if (!ccf.crypto.isValidX509CertBundle(value)) {
+  let check_func = ccf.crypto.isValidX509CertBundle;
+  // Remove after upgrade to 4.x
+  if (check_func === undefined) {
+    check_func = ccf.isValidX509Cert;
```

Comment:
Ah, I missed this during the review! Should be `ccf.isValidX509CertBundle`

Reply:
Doh, yes. This worked in testing because every usage we make of this, including the CA cert bundle, only ever passes one cert. Will fix though.